atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionManager.java

```diff
@@ -294,7 +294,6 @@ public TimestampService getTimestampService() {
 
     public KeyValueServiceStatus getKeyValueServiceStatus() {
         NodeAvailabilityStatus nodeAvailabilityStatus = keyValueService.getNodeAvailabilityStatus();
-        // TODO: if config doesn't match db state return TERMINAL
```

Comment:
I have little context on this, but would it be worth leaving this here in case we actually need to do this in the future?